app/javascript/components/mentoring/mentor-discussion/SolutionInfo.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { TrackIcon } from '../../common/TrackIcon'
+import { Avatar } from '../../common/Avatar'
+import { Student, Track, Exercise } from '../MentorDiscussion'
+
+export const SolutionInfo = ({
+  student,
+  track,
+  exercise,
+}: {
+  student: Student
+  track: Track
+  exercise: Exercise
+}): JSX.Element => {
+  return (
+    <>
+      <TrackIcon title={track.title} iconUrl={track.iconUrl} />
```

Comment:
@iHiD There is a similar block of code in `<Mentoring.Queue />`. Should we extract this?

Reply:
Leave them as different for now. Thanks for checking.